internal/testutil/types.go

```diff
@@ -36,10 +36,6 @@ func MarshalDate(t *time.Time) ([]byte, error) {
 
 func UnmarshalDate(b []byte, t *time.Time) error {
 	// (modified from time.Time.UnmarshalJSON)
```

Comment:
I know this case is likely caught by the template code above now, but if someone calls this public method directly, won't they need these lines as in https://cs.opensource.google/go/go/+/refs/tags/go1.17.1:src/time/time.go;l=1283

Reply:
Do you mean on this style of function in genera, or on this specific one?  In general, folks can include such code if they intend to call the method directly.  In this specific case, the package is internal so no one except us can call it!